packages/did-method-element/docs/local-test.md

```diff
@@ -184,55 +191,74 @@ https://ropsten.infura.io/v3/[YOUR_PROJECT_API_KEY]
-repository before starting the Sidetree-Element service. 
+repository before starting the Sidetree-Element service. Specifically we want
+to set `MONGO_DB_CONNECTION_STRING` and `ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI`
+to use the services running on localhost. For `ETHEREUM_RPC_URL` we want to 
+provide our API url and key. And on `ETHEREUM_MNEMONIC` we will need to provide
+the mnemonic phrase for a minimally funded wallet to use the service. 
 
 ```
 $ cd /home/ubuntu/sidetree.js/packages/dashboard
-$ cp .env.example .env.local
-$ vim .env.local
---- Edit The Following Lines ---
-NEXT_PUBLIC_SIDETREE_METHOD="elem:ropsten"
-NEXT_PUBLIC_OPERATOR=[YOUR_ORGANIZATION_NAME]
-NEXT_PUBLIC_METHOD="Element Ropsten"
-NEXT_PUBLIC_DESCRIPTION="Sidetree on Ethereum Ledger and IPFS Cas."
+$ cp .env.ropsten.example .env.ropsten
+$ vim .env.ropsten
+--- Edit to Resemble the following ---
+SIDETREE_METHOD='elem:ropsten'
 
+# Sidetree Variables
+MONGO_DB_CONNECTION_STRING="mongodb://localhost:27017/"
 DATABASE_NAME="element-ropsten"
+MAX_CONCURRENT_DOWNLOADS=20
+BATCH_INTERVAL_IN_SECONDS=5
+OBSERVING_INTERVAL_IN_SECONDS=5
 
+# Element Node Variables
+ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI='/ip4/127.0.0.1/tcp/5001'
 ELEMENT_ANCHOR_CONTRACT="0x920b7DEeD5CdE055260cdDBD70C000Bbd5b30997"
-ETHEREUM_RCP_URL=https://ropsten.infura.io/v3/[YOUR_PROJECT_API_KEY]
-ETHEREUM_MNEMONIC=[YOUR_WALLET_MNEMONIC]
+ETHEREUM_RPC_URL='https://ropsten.infura.io/v3/[YOUR_PROJECT_API_KEY]'
+ETHEREUM_PROVIDER=$ETHEREUM_RPC_URL
+ETHEREUM_MNEMONIC='[YOUR_MNEMONIC_PHRASE]'
 --- End Contents
 ```
 
-## Start Required Services
-
-IPFS and MongoDB should be running in the background as a daemon service.
-The two services that you will need to run are `ganache-cli` and the
-`element` dashboard. 
+We can then start the element node by running the following command.
 
-You can run these directly in the terminal, or start them with a screen,
-or other method, to have them run in the background, if you choose to
-use it. This guide will provide the commands for running in an active
-terminal.
-
-**Start Element**
 ```
-$ cd packages/dashboard
-$ npm run dev:elem
+$ npm run dev:ropsten
 ```
 
-**Note**: If you are not planning on changing anything inside the UI or the
-API, you might considder buildind and running the Element Service
+## Run Create / Resolve Operation
 
-```
-$ npm run build
-$ npm run start
-```
+Once we have Element running on our local machine, we can then run a test
+operation to see if we are able to create and then resolve a did. 
+
+![Screenshot from 2021-12-14 22-46-41](https://user-images.githubusercontent.com/86194145/146037402-ae75eb83-0394-40f2-bd0e-ecfc090aed29.png)
+
+We can start by opening up `localhost:3000` in our browser and then clicking on the `Wallet` link on the "Manage" card. 
+
+![Screenshot from 2021-12-14 22-47-22](https://user-images.githubusercontent.com/86194145/146037619-c62917e7-a387-40c5-8207-2e8123302627.png)
```

Comment:
consider renaming screenshots

Reply:
Sure, I'll change the alt text to describe the operation.